src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/lib/actions/change_view_flyout.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+import { NotificationsStart } from 'src/core/public';
+import { DashboardPanelState } from 'src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public';
+
+import { EuiFlyout, EuiFlyoutBody, EuiFlyoutHeader, EuiTitle } from '@elastic/eui';
+
+import { IContainer } from '../../../../../../embeddable_api/public/np_ready/public/lib/containers';
+import {
+  IEmbeddable,
+  EmbeddableInput,
+  EmbeddableOutput,
+} from '../../../../../../embeddable_api/public/np_ready/public/lib/embeddables';
```

Comment:
```suggestion
} from '../../../../../../embeddable_api/public/np_ready/public';
```

plugins are only allowed to import top level exports from other plugins, so I think you will need to clean up a couple of the import paths to get past the type errors

Reply:
Done.